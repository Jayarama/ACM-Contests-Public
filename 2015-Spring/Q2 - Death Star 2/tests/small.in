[[n, e], [e, k], [k, m]]
[[w, a], [a, k], [k, d]]
[[b, o], [o, i], [i, c], [c, w]]
