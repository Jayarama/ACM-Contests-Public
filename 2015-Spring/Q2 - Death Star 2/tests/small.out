[g, y], [y, t], [t, o], [o, e], [e, j]
